modules/glib/2.82.2.bcr.2/overlay/gio/BUILD.bazel

```diff
@@ -0,0 +1,713 @@
+load("@bazel_skylib//rules:expand_template.bzl", "expand_template")
+load("@bazel_skylib//rules:run_binary.bzl", "run_binary")
+
+expand_template(
+    name = "gnetworking_h_in",
+    out = "gnetworking.h",
+    substitutions = {
+        "@NAMESER_COMPAT_INCLUDE@": "",
+    },
+    template = "gnetworking.h.in",
+)
+
+[
+    genrule(
```

Comment:
Is it possible to use `run_binary` instead of `genrule` here? The `run_binary` rule doesn't do shell argument parsing, so its behavior is more predictable.